src/bin/todo/update.rs

```diff
@@ -1,8 +1,39 @@
+use chrono::NaiveDateTime;
 use clap::Parser;
+use mindmap::Priority;
 
 #[derive(Parser)]
-pub struct Args {}
+pub struct Args {
+    #[clap(long)]
+    description: String,
+    #[clap(long)]
+    new_description: String,
+    #[clap(long)]
+    new_priority: Priority,
+    #[clap(long, value_parser=mindmap::parse_datetime)]
+    new_deadline: NaiveDateTime,
```

Comment:
Let's keep the deadline as `NaiveDate`; in hindsight, I don't see the time being useful for this since that's too much micro-management

Reply:
Sure